pkg/model/crd.go

```diff
@@ -1855,16 +1858,6 @@ func (r *CRD) goCodeSetOutputReadMany(
 	// operation by checking for matching values in these fields.
 	matchFieldNames := r.listOpMatchFieldNames()
 
-	//  if len(resp.CacheClusters) == 0 {
```

Comment:
If `len(resp.CacheClusters) == 0`, the `for` loop won't run at all, meaning `found` will be `false` and there is a return statement after the `for` loop that returns the same answer if `found == false`. So, behavior shouldn't change when we remove this if block.